### DISCUSSION TITLE
Can it used in vue2 please?

### DISCUSSION THREAD

**Discussion by chenbimo** [joseph2/vue-slide-unlock#2] [Q&A] [answered]
Good project it is.
can it used in vue2 please?

**Comment by j2only** [comment, marked as answer]
I don't know, maybe removing "defineComponent" will help launch it on Vue 2. My goal was to create a component specifically for Vue 3.  By the way, I know a similar component for Vue 2 - https://github.com/AshleyLv/vue-drag-verify.
